plaid/categories.go

```diff
@@ -10,14 +10,14 @@ type Category struct {
 	Hierarchy  []string `json:"hierarchy"`
 }
 
-type CategoryResponse struct {
+type GetCategoryResponse struct {
 	APIResponse
 	Categories []Category `json:"categories"`
 }
 
 // GetCategories returns information for all categories.
 // See https://plaid.com/docs/api/#category-overview.
-func (c *Client) GetCategories() (resp CategoryResponse, err error) {
+func (c *Client) GetCategories() (resp GetCategoryResponse, err error) {
```

Comment:
nit: `GetCategoriesResponse` instead of `GetCategoryResponse`

Reply:
has been resolved already ⚡️